internal/pkg/aws/sessions/sessions.go

```diff
@@ -155,6 +169,13 @@ func (p *Provider) defaultSession() (*session.Session, error) {
 	if err != nil {
 		return nil, err
 	}
+	if _, err = p.sessionValidator.ValidateCredentials(sess); err != nil {
+		if isCredRetrievalErr(err) {
+			return nil, &errCredRetrieval{profile: "profile", parentErr: err}
```

Comment:
Why are we setting this to `"profile"`? In this scenario, shouldn't it be:
```suggestion
			return nil, &errCredRetrieval{parentErr: err}
```
So that the default notice is shown.

Reply:
Oops I misplaced this profile thing here. I have removed it now.